src/components/ResourceForm.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+
+export function ResourceForm() {
```

Comment:
Might be worth renaming these from `<x>Form` to `x<Element>` or something like that, saving the `<x>Form` naming system for the outermost `<form>` component. @radamson thoughts?

Reply:
So this would go from `ResourceForm` to `ResourceElement` or `ResourceGroup` or the like

Reply:
Yeah `ResourceForm` wasn't the best name, but I actually did away with that component in `json-xml-validation` because I created a new component called `FormInputItem` to represent the input-field-or-file-upload items (for both the resource and profile)